Bugzilla/App/BMO/NewRelease.pm

```diff
@@ -179,4 +166,18 @@ sub _add_value {
   }
 }
 
+# Helper function to return a fully formatted version or milestone
+sub _get_formatted_value {
+  my ($product, $value, $type) = @_;
+  use Bugzilla::Logging;
+  DEBUG("product: $product");
```

Comment:
Development debugging lines? If we do want to log this we should import `Bugzilla::Logging` at the top.

Reply:
Ugh. Totally left that there by mistake. Pushing change to remove.